ProjectEureka/project-eureka-flutter
# PE-46| Extracted AppBar And Make it A Component

**What I did:**

- Created a custom AppBar() for Project Eureka
- Our Appbar has the following custom parameters: title, actions, and appBar.
- Our Appbar has the following static parameters: centerTitle, toolbarHeight, and backgroundColor.
- I replaced flutters built in appbar with our custom appbar, in all of our pages that use **appbar:** except for home_page.dart
- Our app bar has a universal backgroundColor.

**How to Test:**

1.  git pull from branch
2. Run flutter pub get
3. Test to make sure all pages I changed have the look of our **EurekaAppbar()**, and what we agreed on figma
4. Test to make sure our backgroundColor can change across the entire app that uses EurekaAppbar(). By changing it within our eureka_appbar.dart under components file. **Don't forget to change this back!**
5. Test to make sure the buttons within the app bar work.
